### PR TITLE
More prime migration, plus other fixes

### PR DESCRIPTION
**Pull request by rpiaggio**
- I was using an Accordion for brightness table when a Panel sufficed.
- ITC errors in the table tooltips didn't show a newline.

### CODE REVIEW

File: explore/src/main/scala/explore/config/sequence/GeneratedSequenceTables.scala [hunk 1]
```diff
@@ -22,20 +21,18 @@ object GeneratedSequenceTables:
   private val component =
     ScalaFnComponent
       .withHooks[Props]
-      .render { props =>
-        <.div(^.height := "100%", ^.overflow.auto)(
```
Comment (cquiroz) [inline explore/src/main/scala/explore/config/sequence/GeneratedSequenceTables.scala:26]: Nice there are done now in css?

File: project/Settings.scala [hunk 2]
```diff
@@ -323,6 +324,12 @@ object Settings {
       Seq("edu.gemini" %%% "refined-algebra" % refinedAlgebra)
     )
 
+    val ScalaCollectionContrib = Def.setting(
+      deps(
+        "org.scala-lang.modules" %%% "scala-collection-contrib"
```
Comment (cquiroz) [inline project/Settings.scala:329]: What are we using this for? I thought this mostly needed for cross built libraries

**Review by cquiroz — APPROVED** [review]
LGTM

File: explore/src/main/scala/explore/config/sequence/GeneratedSequenceTables.scala (hunk 1 above)
Comment (rpiaggio) [reply explore/src/main/scala/explore/config/sequence/GeneratedSequenceTables.scala:26]: Turns out it's not necessary anymore. I assume it was in the migration to Prime.

File: project/Settings.scala (hunk 2 above)
Comment (rpiaggio) [reply project/Settings.scala:329]: For `list.intersperse(...)`.

**Pull request merged**